src/tree/contexts/ContextTreeItem.ts

```diff
@@ -0,0 +1,61 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzExtParentTreeItem, AzExtTreeItem, IActionContext } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+import { dockerContextManager } from "../../utils/dockerContextManager";
+import { getThemedIconPath, IconPath } from '../IconPath';
+import { LocalContextInfo } from "./LocalContextInfo";
+
+export class ContextTreeItem extends AzExtTreeItem {
+    public static contextValue: string = 'context';
+    public contextValue: string = ContextTreeItem.contextValue;
+    private readonly _item: LocalContextInfo;
+
+    public constructor(parent: AzExtParentTreeItem, item: LocalContextInfo) {
+        super(parent);
+        this._item = item;
+    }
+
+    public get createdTime(): number {
+        return this._item.createdTime;
+    }
+
+    public get label(): string {
+        return ext.contextsRoot.getTreeItemLabel(this._item);
+    }
+
+    // this is the description shown in tree item which can include one or more properites combined.
+    // This is not the description of the context.
```

Comment:
Can the user configure it to be the actual context description? If so I'd say not _necessarily_ the description.

Reply:
Yes user can still configure to see the contest description. It is in ContexsTreeItem.getPropertyValue